tortoise/queryset.py

```diff
@@ -679,9 +679,7 @@ def _make_query(self) -> None:
             self.query._offset = self._offset
         if self._distinct:
             self.query._distinct = True
-        self.resolve_ordering(
-            self.model, self.model._meta.basetable, self._orderings, self._annotations
-        )
+        print(str(self.query))
```

Comment:
Print debugging for the Win!

Reply:
Sorry,that's my time to get off work and lose checking...😅

Reply:
We all do it, don't worry about it. :+1: